module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -194,6 +194,7 @@ namespace sys
         // Power off request (pending)
         PowerOff();
 
+        // TODO these were globals = never cleared, is it needed now?
```

Comment:
If their lifetime is finally managed by RAII then removing them explicitly is not required anymore. One thing to keep in mind is the order in which they will be constructed/destroyed.

Reply:
oh, that was a note to me. I spend some time tinkering why such choices were made but at the moment I don't know if it was a deliberate decision or just a copy of the approach someone had before.